### PR TITLE
Build Debian packages on push to master

### PR DESCRIPTION
**Pull request by qwandor**
Also a simple script to do it locally.

This has to work around https://github.com/mmstick/cargo-deb/issues/151, which is a bit annoying.

### CODE REVIEW

File: .github/workflows/package.yml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      id: cache-cargo-deb
+      uses: actions/cache@v2
+      with:
+        path: |
+          ~/.cargo/bin/cargo-deb
+          ~/.cargo/bin/cross
+          ~/.cargo/.crates.toml
+          ~/.cargo/.crates2.json
+        key: ${{ runner.os }}-cargo-deb-${{ env.cargo-deb-version }}
+
```
Comment (alsuren) [inline .github/workflows/package.yml:82]: cache target as well?

**Review by alsuren — APPROVED** [review]
- [ ] merge and then check that the debian package works on the raspberry pis
- [x] Make a task for the auto-dep warning

File: .github/workflows/package.yml (hunk 1 above)
Comment (qwandor) [reply .github/workflows/package.yml:90]: Whoops, yes.

**Comment by qwandor** [comment]
This works towards #18.

**Pull request merged**